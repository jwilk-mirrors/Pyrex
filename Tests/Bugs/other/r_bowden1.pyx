def f(x):
	cdef unsigned long long ull
	ull = x
	return ull + 1

def g(unsigned long x):
	return x + 1
