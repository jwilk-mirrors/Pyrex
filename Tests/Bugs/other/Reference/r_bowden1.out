101
3000000001
